docs/x509.rst

```diff
@@ -166,6 +166,180 @@ X.509 Certificate Object
             >>> cert.not_valid_after
             datetime.datetime(2030, 12, 31, 8, 30)
 
+    .. attribute:: issuer
+
+        .. versionadded:: 0.8
+
+        :type: :class:`Name`
+
+        The :class:`Name` of the issuer.
+
+        :raises cryptography.x509.UnknownAttribute: If there is an unknown
+            attribute in the issuer.
+
+    .. attribute:: subject
+
+        .. versionadded:: 0.8
+
+        :type: :class:`Name`
+
+        The :class:`Name` of the subject.
+
+        :raises cryptography.x509.UnknownAttribute: If there is an unknown
+            attribute in the subject.
+
+
+.. class:: Name
+
+    .. versionadded:: 0.8
+
+    An X509 Name is an ordered list of attributes. The entire list can be
+    obtained with :attr:`attributes` or you can use the helper properties to
+    obtain the specific type you want.
+
```

Comment:
Should we mention that they're sometimes rendered with the `CN=abc/OU=abc/O=whatever` format?